.github/workflows/release.yml

```diff
@@ -79,17 +77,13 @@ jobs:
           cache: npm
       - name: Install dependencies
         run: npm ci
-      - name: Execute version target on all affected packages and release them
-        env:
-          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
-        run: npx nx version @code-pushup/cli-source
-      - name: Deploy to NPM
+      - name: Execute version target on all affected packages, release then publish them
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        # un-exclude packages when they're ready for public release
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
         run: |
           echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
-          npx nx run-many -t=deploy --exclude=plugin-lighthouse,nx-plugin
+          npx nx release
```

Comment:
Just be aware that this might fail as Nx [recommends](https://nx.dev/recipes/nx-release/get-started-with-nx-release#create-the-first-release) to run the first release with their tool using the `--first-release` flag.

Reply:
Good to know. Will run with `--first-release` as soon as this is merged :slightly_smiling_face: